checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -2025,14 +2028,19 @@ private void checkMustCall(
 
       // sometimes the store is null!  this looks like a bug in checker dataflow.
       // TODO track down and report the root-cause bug
-      CFValue aliasCFValue = cmStore != null ? cmStore.getValue(alias.reference) : null;
+      AccumulationValue aliasCFValue = cmStore != null ? cmStore.getValue(alias.reference) : null;
```

Comment:
Do we know why this variable is called `aliasCFValue`, in particular what `CF` stands for?  If not, maybe we should rename?

Reply:
I think "CF" there is based on the old type (`CFValue`). I've changed the name of the variable to `cmValue`, which parallels `cmAnno` on the next line (both are for `alias`, which is another local, but I think that's 1) implied from context, and 2) not a great name itself.